akka-actor/src/main/scala/akka/actor/FSM.scala

```diff
@@ -315,6 +315,10 @@ trait FSM[S, D] extends Actor with Listeners with ActorLogging {
   val Event: FSM.Event.type = FSM.Event
   val StopEvent: FSM.StopEvent.type = FSM.StopEvent
 
+  object → {
```

Comment:
is this really binary backwards compatible? if it would have been a `def` it wouldn't